velox/docs/functions/spark/datetime.rst

```diff
@@ -93,6 +93,12 @@ These functions support TIMESTAMP and DATE input types.
         SELECT from_unixtime(3600, 'yyyy'); -- '1970'
         SELECT from_unixtime(9223372036854775807, "yyyy-MM-dd HH:mm:ss");  -- '1969-12-31 23:59:59'
 
+.. spark:function:: from_utc_timestamp(timestamp, string) -> timestamp
+
+    Returns the timestamp value from UTC timezone to the given timezone. ::
+
+        SELECT from_utc_timestamp('2015-07-24 07:00:00', "America/Los_Angeles"); -- '2015-07-24 00:00:00'
```

Comment:
use single quotes around time zone (double quotes in SQL are used for column names)

Reply:
Updated, thanks!